src/sentry/api/endpoints/organization_events_facets_performance.py

```diff
@@ -406,35 +402,10 @@ def query_facet_performance(
             turbo=sample_rate is not None,
             limit=limit,
             offset=offset,
-            limitby=["tags_key", tag_key_limit] if not tag_key else None,
+            limitby=("tags_key", tag_key_limit) if not tag_key else None,
         )
     translated_aggregate_column = tag_query.resolve_column(aggregate_column)
 
-    if include_count_delta:
-        middle = snuba_params.start_date + timedelta(
-            seconds=(snuba_params.end_date - snuba_params.start_date).total_seconds() * 0.5
-        )
-        middle = datetime.strftime(middle, DateArg.date_format)
-
-        count_range_1 = tag_query.resolve_function(
-            f"count_range(lessOrEquals, {middle})", overwrite_alias="count_range_1"
-        )
-        count_range_total = tag_query.resolve_function(
-            "count()",
-            overwrite_alias="count_range_total",
-        )
-
-        count_delta = tag_query.resolve_division(
-            Function(
-                "minus", [Function("minus", [count_range_total, count_range_1]), count_range_1]
-            ),
-            count_range_1,
-            "count_delta",
-        )
-
-        tag_query.columns.extend([count_range_1, count_range_total, count_delta])
-        tag_query.aggregates.extend([count_range_1, count_range_total, count_delta])
-
```

Comment:
this is dead code -- no caller passes this arg.  the type checker pointed out that `resolve_division` would always AttributeError

last reference to that arg removed in 5a34e6e307db9b96185f0959263761f524c5f7ac